MinPlatformPkg/Test/Library/PlatformConfigCheckLibNull/DxePlatformConfigCheckLibNull.inf

```diff
@@ -0,0 +1,28 @@
+## @file
+# Platform configuration check library NULL implementation for TestPointCheckLib
+#
+# Copyright (c) Microsoft Corporation. All rights reserved
+#
+##
+
+[Defines]
+  INF_VERSION                    = 1.27
+  BASE_NAME                      = DxePlatformConfigCheckLib
+  FILE_GUID                      = ABA05EC8-A7C3-4198-B8ED-4A0D6B9F8296
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = DxePlatformConfigCheckLibNull
+
+[LibraryClasses]
+  BaseLib
+  DebugLib
+  UefiLib
+  PrintLib
+
+[Packages]
+  MinPlatformPkg/MinPlatformPkg.dec
+  MdePkg/MdePkg.dec
+  MdeModulePkg/MdeModulePkg.dec
```

Comment:
For how simple this NULL instance is, try to drop the MdeModulePkg dependency if possible.